internal/testutil/db.go

```diff
@@ -382,6 +382,10 @@ func (c *DBContext) Import(ctx context.Context, r io.Reader) error {
 
 		// Populate sequences
 		for _, seq := range c.D.Sequences() {
+			if !containsTable(c.Schema, seq.Table) {
```

Comment:
I don't understand why this is needed. And, if really needed, it should not live in this commit/PR.

Reply:
I think it is needed because the test TestCreate and TestMigrate fail with this error `failed to update sequence for run_sequence_seq: no such table: commitstatusdatadelivery`.
This method update the sequence values based by the current db version(version 3 for notification), but the creation of the db schema is done out by the Create method, that different db version based of create fixtures.

I open a new PR to fit it.

Reply:
You discovered an issue but then created a workaround.
Just fix the DBContext.Import to use the right sequences for that db version from createData.Sequences

Reply:
I opened PR #441 to fix it